src/NUnitFramework/nunitlite.runner/AutoRun.cs

```diff
@@ -102,9 +102,9 @@ public int Execute(string[] args)
             }
 #endif
 
-            _textUI.DisplayTestFiles(new string[] { callingAssembly.GetName().Name });
             _textUI.DisplayRuntimeEnvironment();
-            _textUI.DisplayRequestedOptions();
+            _textUI.DisplayTestFiles(new string[] { callingAssembly.GetName().Name });
+//            _textUI.DisplayRequestedOptions();
```

Comment:
That was my intent. I got confused by the several copies of the file. I think I'll move the two varying files to a subdirectory.